lib/plugins/HtmlWebpackNewRelicPlugin.js

```diff
@@ -0,0 +1,73 @@
+;NREUM.loader_config={accountID:"${this.options.accountID}",trustKey:"${this.options.trustKey}",agentID:"${this.options.agentID}",licenseKey:"${this.options.licenseKey}",applicationID:"${this.options.applicationID}"}
+;NREUM.info={beacon:"bam-cell.nr-data.net",errorBeacon:"bam-cell.nr-data.net",licenseKey:"${this.options.licenseKey}",applicationID:"${this.options.applicationID}",sa:1}`;
+
+    Object.keys(this.options).forEach((key) => {
+      if (!this.options[key]) {
+        throw new Error(`${key} argument is required`);
+      }
+    });
+  }
+
+  apply(compiler) {
+    compiler.hooks.compilation.tap('HtmlWebpackNewRelicPlugin', (compilation) => {
+      HtmlWebpackPlugin.getHooks(compilation).alterAssetTagGroups.tap(
+        'HtmlWebpackNewRelicPlugin',
+        (data) => {
+          const newRelicScriptTag = HtmlWebpackPlugin.createHtmlTagObject(
+            'script',
+            { type: 'text/javascript' },
+            this.newrelicstring,
+          );
+          data.bodyTags.push(newRelicScriptTag);
```

Comment:
If possible, I think we might want to use `data.headTags` here to add this NR script to the `head` vs. the `body` per the recommend of NR ([source](https://docs.newrelic.com/docs/browser/new-relic-browser/page-load-timing-resources/instrumentation-browser-monitoring/#javascript-placement)).